src/components/Carousel.tsx

```diff
@@ -0,0 +1,88 @@
+import { Link } from "react-router-dom";
+import arrow from "../assets/arrow.svg";
+import { useState } from "react";
+import { motion } from "framer-motion";
+interface CarouselProps {
+  articles: Article[];
+  to: string;
+}
+export default function Carousel({ articles, to }: CarouselProps) {
```

Comment:
The name carousel by itself it's not descriptive, choose a name that describes the type of content it's displaying